lib/DDG/Spice/GoWatchIt.pm

```diff
@@ -0,0 +1,36 @@
+package DDG::Spice::GoWatchIt;
+
+use DDG::Spice;
+
+name "GoWatchIt Search";
+source "GoWatchIt.com";
+description "Find out where to watch your favorite movies and shows!";
+category "entertainment";
+topics "entertainment";
+code_url "https://github.com/plexusent/zeroclickinfo-spice/blob/gwi/lib/DDG/Spice/GoWatchIt.pm";
```

Comment:
Should be `https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/lib/DDG/Spice/GoWatchIt.pm`